website/management/commands/check_keywords.py

```
@@ -6,38 +10,219 @@
+
+            # Block private IP ranges (RFC 1918, link-local, localhost, reserved)
+            if ip_obj.is_private or ip_obj.is_loopback or ip_obj.is_link_local or ip_obj.is_reserved:
+                self.stderr.write(f"    Blocked private/reserved IP: {ip} for {hostname}")
+                return False
+
+            # Block cloud metadata endpoints
+            if ip == "169.254.169.254":
+                self.stderr.write(f"    Blocked cloud metadata endpoint: {ip}")
+                return False
+
+            return True
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)